src/api/subscriptions.js

```diff
@@ -271,6 +272,40 @@ export class Subscriptions {
    */
   getPropensityModule() {}
 
+  /**
+   * Returns true if the Google app login flow is currently
+   * active, i.e., if googleAppLoginFlowComplete should be
+   * called when the user logs in.
+   *
+   * @return {boolean}
+   */
+  isGoogleAppLoginFlowActive() {}
```

Comment:
Nit: active or available?

Reply:
"Active", in that this is the login page running in a CCT that was launched by the app login flow.